clients/vscode/src/chat/WebviewHelper.ts

```diff
@@ -576,13 +575,14 @@ export class WebviewHelper {
         };
         const smartApplyInEditor = async (editor: TextEditor, opts: { languageId: string; smart: boolean }) => {
           if (editor.document.languageId !== opts.languageId) {
-            getLogger().info("editor:", editor.document.languageId, "opts:", opts.languageId);
+            this.logger.debug("Editor's languageId:", editor.document.languageId, "opts.languageId:", opts.languageId);
             window.showInformationMessage("The active editor is not in the correct language. Did normal apply.");
             applyInEditor(editor);
             return;
           }
 
-          getLogger("Tabby").info("Smart apply in editor started.", content);
+          this.logger.info("Smart apply in editor started.");
+          this.logger.trace("Smart apply in editor with content:", { content });
```

Comment:
Avoid logging a message that is too long to the `info` level logs.
I changed `content` to `trace` level instead.